app/subscriber/src/components/user-profile/UserProfile.tsx

```diff
@@ -1,6 +1,7 @@
 import { ScreenSizes } from 'components/layout/constants';
+import React, { useState } from 'react';
```

Comment:
Just have to remove useState for the linter!

Reply:
Thanks. I found my linter was not working, I am going fix it.